CITATION.cff

```diff
@@ -179,6 +179,10 @@ authors:
       family-names: Welzel
       affiliation: 'Department of Neurology, Kiel University, Germany'
       orcid: 'https://orcid.org/0000-0003-4782-5360'
+    - given-names: Kaare
+      family-names: Mikkelsen
+      affiliation: 'Department of Electrical and Computer Engineering, Aarhus University, Denmark'
+      orcid: 'https://orcid.org/0000-0002-7360-8629'
```

Comment:
@kaare-mikkelsen, please check this and let me know if we need to correct something. 🙏

Reply:
it is correct, thank you :)